### PR TITLE
 Replace prebuilt KTX, adjust default intensities.

### PR DESCRIPTION
**Pull request by prideout**
Before / after screenshots.  Note that auto-adjustment has been moved into a button.

before_after

### CODE REVIEW

**Comment by prideout** [comment]
NOTE: Since this removes our prebuilt KTX files, some of the Android samples will need to be fixed up.

File: libs/gltfio/include/gltfio/SimpleViewer.h [hunk 1]
```diff
@@ -144,9 +149,9 @@ class SimpleViewer {
     // Properties that can be changed from the UI.
     int mCurrentAnimation = 1;
     bool mResetAnimation = true;
-    float mIblIntensity = 30000.0f;
+    float mIblIntensity = 15000.0f;
     float mIblRotation = 0.0f;
-    float mSunlightIntensity = 100000.0f;
+    float mSunlightIntensity = 70000.0f;
```
Comment (romainguy) [inline libs/gltfio/include/gltfio/SimpleViewer.h]: Those are low values for the default camera setup

**Comment by prideout** [comment]
Discussed with Mathias; our sample apps currently look very bright with IBL's that have baked-in sunlight.  Alternatively we could use a different default IBL that doesn't have a sun.

I'll look into Gradle enhancements and add to the PR.

**Comment by romainguy** [comment]
They look very bright when the directional light is automatically extracted from the IBL, they were fine before.

**Comment by pixelflinger** [comment]
@romainguy the other change is that "before" the IBLs where clamped, effectively removing the sun.  But now that they're not clamped, whether we use the default sun intensity or the calculated one, we end-up too bright.  The crux of the issue is that we should follow our own rules and remove the sun from the IBLs.

**Comment by romainguy** [comment]
Except that's only true of IBLs that have a large dynamic range and from what I've seen, they're not super common. It also means that now the default dynamic light intensity has lost its physical meaning.

**Comment by prideout** [comment]
Hmm, we can discuss further tomorrow.  If it helps here's another example, before / after shots for the cmgen commit at 4d7ab6547c7374ddb067af3980b64772bdd6f449

before
after

**Comment by prideout** [comment]
Closing this PR since it conflates some non-controversial changes (removing pre-built KTX) with controversial changes (tweaking the defaults)

**Pull request closed without merging**